### ISSUE TITLE
Collaboration Request

### ISSUE THREAD

**Issue by gaotongxiao** [JiaquanYe/MASTER-mmocr#6]
Hi authors, I just came into this project and found it written well! I wonder if you would like to raise a PR to MMOCR so that more researchers could benefit from your work? I'm also happy to chat if you have any other thoughts :)

**Comment by JiaquanYe** [comment]
> Hi authors, I just came into this project and found it written well! I wonder if you would like to raise a PR to MMOCR so that more researchers could benefit from your work? I'm also happy to chat if you have any other thoughts :)

It's my pleasure to do this. I will raise a PR within these few days.

**Comment by gaotongxiao** [comment]
Awesome! Let me know when you need any assistance.

**Comment by gaotongxiao** [comment]
Hi, are you still interested in raising a PR? If you need any assistance from us (and the community), just mail to mmocr@openmmlab.com and I’ll get you in our Slack group. We can help you organize the project and assign some tasks to others etc.

**Comment by JiaquanYe** [comment]
> Hi, are you still interested in raising a PR? If you need any assistance from us (and the community), just mail to mmocr@openmmlab.com and I’ll get you in our Slack group. We can help you organize the project and assign some tasks to others etc.

Of course, I am still interested in PR. I have reimplemented the project in the newest mmocr version. I use mmocr official datasets to run some experiments and confirm the performance. It takes some weeks causes of the computing resources limitations. The good news is the experiments almost done.
